Server/Quality_of_Life_changer.WebApi/Controllers/AuthController.cs

```diff
@@ -65,8 +65,12 @@ public async Task<IActionResult> GetAllUsers()
         return response == null ? NotFound() : Ok(response);
     }
 
-    private UserModel MapUserToUserModel(QolcUser user)
+    [Route("TodayEvents")]

```

Comment:
routes should be named in `barbeque-case`.
please move calendar-related endpoint to a separate (Calendar) controller.
also please follow REST rules for routes naming.